src/main/php/org/bovigo/vfs/vfsStreamAbstractContent.php

```diff
@@ -148,7 +148,7 @@ public function getType()
      * sets the last modification time of the stream content
      *
      * @param   int  $filemtime
-     * @return  vfsStreamContent
+     * @return  $this
```

Comment:
According to http://www.phpdoc.org/docs/latest/references/phpdoc/types.html `$this` is not a valid type, only self could be used. But in this case it would be interpreted as `vfsStreamAbstractContent`, and I'm not particularly happy about that as the abstract type should not be visible in the API.
I'm not sure if `self` would help with PhpStorm then, but I'd rather follow a standard then to change it to something IDE specific. Maybe it would be a better idea to simply use fully qualified class names in this places?